docs/index.html

```diff
@@ -64,10 +70,8 @@
           '/': 'Search'
         }
       },
-      darkMode: {
-        light: {
-          toggleBtnBg: '#42b983'
-        }
+      darklightTheme: {
+        bodyFontSize : '15px',
```

Comment:
whats the default ?

Reply:
the default font size of docsify site is 15 px. In my plugin I gave 17px as default. So I am overriding here.